cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -1016,6 +1016,28 @@ public static final class Dataset {
       public static final String LEVELDB_STATS_REPORT_INTERVAL = "metrics.dataset.leveldb.stats.report.interval";
     }
 
+    /**
+     * Namespaced Service Accounts metrics related constants.
+     */
+    public static final class Credential {
+      public static final String NAMESPACE_CREDENTIALS_WORKLOAD_IDENTITY_VALIDATE_ATTEMPTS_COUNT
```

Comment:
Instead of having a unique metric between attempts and successes, should we report whether the metric was successful or not via a tag?